.github/workflows/playwright.yml

```diff
@@ -0,0 +1,44 @@
+name: Playwright Tests
+on:
+  push:
+    branches-ignore:
+      - "main"
+jobs:
+  test:
+    strategy:
+      matrix:
+        node-version: [17.x, 18.x, 19.x]
+    timeout-minutes: 60
+    runs-on: macos-latest
+    steps:
+      - name: "Checkout"
+        uses: actions/checkout@v3
+
+      - name: Install pnpm
+        uses: pnpm/action-setup@v2.2.4
+
+      - name: Build with NodeJS ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: "pnpm"
+
+      - name: Install deps and build
+        run: pnpm i && pnpm run build
+
+      - name: Install again to link deps
+        run: pnpm i
+
+      - name: Install playwright browsers
+        run: pnpm exec playwright install
+
+      - name: Run Playwright tests
+        run: pnpm --filter static-site run playwright
+
+      - name: Upload 'React' Playwright Report
```

Comment:
nit: rename to static-site, rather than react

Reply:
I just removed the specification